### PR TITLE
Feature/reporting api

### PR DESCRIPTION
**Pull request by spyder-guy**

### CODE REVIEW

**Review by brent-spyder — CHANGES_REQUESTED** [review]
Make the two environment variable changes and then I think it's good to go

File: spyctl_api/test-docker-compose.yaml [hunk 1]
```diff
@@ -4,6 +4,11 @@ services:
   spyctl_test_api:
     image: spyctl_test_api:latest
     hostname: spyctl_test_api
+    environment:
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+      - AWS_ROLE_ARN=arn:aws:iam::211125369423:role/OrganizationAccountAccessRole
```
[the remark this answers is not in the document]
Comment (spyder-guy) [reply spyctl_api/test-docker-compose.yaml]: ✅

File: spyctl_api/test-docker-compose.yaml [hunk 2]
```diff
@@ -4,6 +4,11 @@ services:
...
+      - REPORT_BUCKET=integration2.reports
```
[the remark this answers is not in the document]
Comment (spyder-guy) [reply spyctl_api/test-docker-compose.yaml]: ✅

**Review by brent-spyder — APPROVED** [review]

File: spyctl_api/app/config.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+import json
+import os
+from typing import Any, Optional
+
+import aioboto3
+
+_config = dict()
+
+
+def deployment_2_bucket(deployment: str) -> str:
```
Comment (brent-spyder) [inline spyctl_api/app/config.py:12]: Not the biggest fan of having the hard coded bit in the python file, but in the interest of getting the feature out I think we'll just live with it.

Could you put a TODO comment here to move this to a loadable config file? Who knows if anyone will ever stand up their own spyctl api service, but I'd like to leave the option open

**Review by brent-spyder — CHANGES_REQUESTED** [review]

File: spyctl_api/app/config.py (hunk 3 above)
Comment (spyder-guy) [reply spyctl_api/app/config.py:12]: done

**Review by brent-spyder — APPROVED** [review]

**Pull request merged**